Kerberos.NET/Server/DefaultRealmSettings.cs

```diff
@@ -10,11 +10,21 @@ public class DefaultRealmSettings : IRealmSettings
     {
         private readonly Krb5ConfigDefaults defaults;
         private readonly Krb5RealmConfig config;
+        private readonly KerberosCompatibilityFlags compatibilityFlags;
 
         public DefaultRealmSettings(Krb5ConfigDefaults defaults, Krb5RealmConfig config)
+            : this(defaults, config, KerberosCompatibilityFlags.None)
+        {
+        }
+
+        public DefaultRealmSettings(
+            Krb5ConfigDefaults defaults,
+            Krb5RealmConfig config,
+            KerberosCompatibilityFlags compatibilityFlags)
```

Comment:
Is this (KerberosCompatibilityFlags) something that could go to Krb5RealmConfig instead so it can be configured via config files? Is that object extensible and would it make sense to move a configuration there ?